src/utils/mp4-tools.ts

```diff
@@ -205,7 +201,7 @@ export function parseVideoSegmentTextTrackSamples (data, videoTrackId) {
   });
 }
 
-export function parseVideoSegmentId3TrackSamples (data) {
+export function parseEmsg (data) {
```

Comment:
Still no support for emsg v1 right?  Can we someone add that to this PR?

https://aomediacodec.github.io/id3-emsg/
"One or more Event Message boxes (‘emsg’) [CMAF] can be included per segment. Version 1 of the Event Message box [DASH] must be used."

Reply:
Not sure if I understand what you're saying? This is v1 right?